core/store.go

```diff
@@ -242,6 +242,26 @@ func Get(k string) *Obj {
 	return getHelper(k, true)
 }
 
+func GetDel(k string) *Obj {
+	var v *Obj
+	withLocks(func() {
+		ptr, ok := keypool[k]
+		if !ok {
+			return
+		}
+
+		v = store[ptr]
+		if v != nil {
+			expired := hasExpired(v)
+			deleteKeyWithOperationName(k, ptr, v, "GETDEL")
```

Comment:
retaining `DEL` is fine as an operation name, the name of the operation should just signify the core operation that caused a change in the keyset - `DEL` is enough for that.

Reply:
Okay. will remove that